cypress/e2e/models/migration/applicationinventory/analysis.ts

```diff
@@ -560,14 +561,21 @@ export class Analysis extends Application {
         clickItemInKebabMenu(this.name, "Cancel analysis");
     }
 
-    verifyMergedLogContain(): void {
+    verifyLogContains(analysisLogView: AnalysisLogView, searchText: string): void {
         this.openAnalysisDetails();
         cy.get(logFilter).eq(2).click();
-        clickByText(logDropDown, "Merged log view");
-
-        // Wait for the editor content to load and assert expected text
-        cy.get(".pf-v5-c-code-editor__code", { timeout: 5000 }).then(($editor) => {
-            expect($editor.text()).to.contain("lspServerName: generic");
-        });
+        clickByText(logDropDown, analysisLogView);
+        cy.wait(3 * SEC);
+
+        cy.get(".pf-v5-c-code-editor__code", { timeout: 10000 })
+            .should("be.visible")
+            .click()
+            .type("{ctrl+f}")
+            .type(`${searchText}`)
+            .type("{enter}")
+            .wait(2 * SEC)
+            .type("{Esc}");
+
+        cy.get(".pf-v5-c-code-editor__code", { timeout: 10000 }).should("be.visible").click();
```

Comment:
I see that you're filtering by the expected text, but you're not verifying that the text is actually there or that the filter returns results, right?

Reply:
Nice catch :D, it was accidently deleted
Thanks!!